mscore/preferences.cpp

```diff
@@ -220,6 +220,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_SCORE_NOTE_PLAYONCLICK,                          new BoolPreference(true, false)},
             {PREF_SCORE_NOTE_DEFAULTPLAYDURATION,                  new IntPreference(300 /* ms */, false)},
             {PREF_SCORE_NOTE_WARNPITCHRANGE,                       new BoolPreference(true, false)},
+            {PREF_SCORE_NOTE_INPUT_DISABLE_MOUSE_INPUT,            new BoolPreference(false, true)},
```

Comment:
and here,
But not sure we'd still need it hidden in Advanced Preferences?

Reply:
Yeah, that could be removed... P.S. apparently forgot to pull from your 3.x before making a branch based on the last PR, so the second commit showed up. Hopefully isn't a problem though, but if it is I can redo it this and then make the preference set to false to not show up...

Reply:
Too late, I've just merged it ;-)